### PR TITLE
Ft/footer section

### PR DESCRIPTION
**Pull request by Fidesnoella**
# Description
Done with Footer to all break point 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent

### CODE REVIEW

File: components/layouts/Footer.jsx [hunk 1]
```diff
@@ -1,10 +1,28 @@
 import React from 'react';
+import Wrapper from '../wrappers/Wrapper';
+import Button from '../../components/ui/Button'
+import Link from 'next/link';
 
 function Footer(props) {
```
Comment (esther921) [inline components/layouts/Footer.jsx]: If you didn't use the props anywhere you can remove it

File: components/layouts/Footer.jsx [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
     return (
-        <div>
-            <h1>Footer</h1>
-        </div>
+        <Wrapper>
+           <div className='flex flex-col sm:flex-row justify-between gap-y-12 sm:gap-y-0 text-xl md:text-2xl font-normal leading-8 tracking-thiner md:tracking-thin'>
+            <div>
+                <ul>
```
Comment (esther921) [inline components/layouts/Footer.jsx]: According to the figma design, this is a paragraph you'll have to give it just a width that it follows, don't make it a list and also remove the break before '10999 Berlin', let it break it's self depending on a width or anything else!

File: components/layouts/Footer.jsx [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+                    <li>Waldemarstraße 38, <br /> 10999 Berlin </li>
+                </ul>
+            </div>
+            <div>
+                <Button link={"https://twitter.com/apeunit"} >Twitter</Button>
+                <Button link={"https://www.facebook.com/apeunit/"} >Facebook</Button>
+                <Button link={"https://www.linkedin.com/company/ape-unit/?originalSubdomain=de"}>LinkedIn</Button>
+            </div>
+            <div>
+                <span><Link href={"https://apeunit.com/impressum.txt"} target="_blank">Impressum</Link></span> / <span><Link href={"https://apeunit.com/datenschutz.txt"} target="_blank">Privacy</Link></span>
```
Comment (esther921) [inline components/layouts/Footer.jsx:20]: To put this forward slash '/', it is better it and even the spans in a paragraph for the '/' not to be a direct content of a div!

**Review by NNesta — APPROVED** [review]
Well Done

**Review by esther921 — APPROVED** [review]
Looks good to me!

**Pull request merged**